### PR TITLE
Waiting List Admin Panel (Readonly)

### PR DESCRIPTION
**Pull request by MelissaAutumn**
To get visibility on the waiting list, I've added a read-only admin panel view. 

image

Bit of a rush, we'll add the "Accept x number of peeps" as a later feature.

### CODE REVIEW

File: frontend/src/views/admin/WaitingListPanelView.vue [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      :loading="loading"
+    >
+    </data-table>
+  </div>
+  <div v-else class="flex size-full min-h-[75vh] items-center justify-center">
+    <loading-spinner/>
+  </div>
+</template>
+
+<script setup>
```
Comment (devmount) [inline frontend/src/views/admin/WaitingListPanelView.vue:34]: Do we want to put the script tag at the top for all (new) single file components?

**Review by devmount — APPROVED** [review]
Thank you! Just some minor questions.

File: frontend/src/views/admin/WaitingListPanelView.vue [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      },
+    ],
+    /**
+     * Callback function, filter the list by selectedKey and return it back to the table
+     * @param selectedKey
+     * @param mutableDataList
+     * @returns {*}
+     */
+    fn: (selectedKey, mutableDataList) => {
+      console.log(mutableDataList)
```
[the remark this answers is not in the document]
Comment (MelissaAutumn) [reply frontend/src/views/admin/WaitingListPanelView.vue:171]: whoops, thanks!

File: frontend/src/views/admin/WaitingListPanelView.vue (hunk 1 above)
Comment (MelissaAutumn) [reply frontend/src/views/admin/WaitingListPanelView.vue:34]: Sure! I think that's probably a good plan.

**Pull request merged**